### PR TITLE
Feature/oui button guide

### PR DESCRIPTION
**Pull request by varun257**
OUI guide component
Migrate cui guide to oui component
Reuse action menu items and wrap it to create guide component

### CODE REVIEW

File: packages/oui-button-guide/button-guide.less [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }
+  .oui-guide__footer {
+    .oui-action-menu__item {
+      .oui-button_action-menu {
+        text-align: center;
+        font-weight: @oui-font-semibold;
+        text-transform: capitalize;
+      }
+    }
+    .oui-button_secondary {
```
Comment (radireddy) [inline packages/oui-button-guide/button-guide.less]: include primary and default buttons also

File: packages/oui-button-guide/button-guide.less [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  .oui-guide__body {
+    max-height: 70vh;
+    overflow-y: auto;
+  }
+
+  //override oui-dropdown size
+  .oui-dropdown__content {
+    width: @oui-guide-dropdown-content-width;
+
+    &_bottom {
```
Comment (radireddy) [inline packages/oui-button-guide/button-guide.less]: should be &-bottom. It's not a modifier.

Comment (varun257) [reply packages/oui-button-guide/button-guide.less]: This is the class in oui-dropdown. overriding this

File: packages/oui-guide-dropdown/guide-dropdown.less [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+@media screen and (max-width: @oui-responsive-breakpoint-phone) {
+  .oui-guide {
+    &__button {
+      &-title {
+        display: none;
+      }
+    }
+  }
+}
```
Comment (neolitec) [inline packages/oui-guide-dropdown/guide-dropdown.less]: Blank line at the end of the file. IDEs can be setup for that, automatically if you use EditorConfig.

**Comment by neolitec** [comment]
I don't think this component respect the UI specs. In addition, there is some UI problems in spacings:
image

File: packages/oui-guide-dropdown/guide-dropdown.less (hunk 3 above)
Comment (varun257) [reply packages/oui-guide-dropdown/guide-dropdown.less]: The spacing was due to a wrong class that was added. Have removed it and it should work properly now.

**Comment by AxelPeter** [comment]
The component is now prioritized by CX and will be taken into account by the UI Kit team
Thanks for your contribution !

**Pull request closed without merging**